scripts/storybook/webpack.config.js

```diff
@@ -15,18 +15,34 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.(scss|css)$/,
-        exclude: /node_modules/,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: [
-            'css-loader',
-            'resolve-url-loader',
-            {
-              loader: 'sass-loader',
+        test: /\.stories\.js$/,
+        loaders: [
+          {
+            loader: require.resolve('@storybook/addon-storysource/loader'),
+            options: {
+              prettierConfig: {
+                parser: 'babylon',
+              },
             },
-          ],
-        }),
+          },
+        ],
+        enforce: 'pre',
+      },
+      {
+        test: /\.s?[ac]ss$/,
```

Comment:
Not sure if `/\.(sa|sc|c)ss$/` wouldn't be safer - not sure any of those loaders have `.ass` file support.

Reply:
will change it